app/controllers/registrations_controller.rb

```diff
@@ -26,6 +26,15 @@ def disable_two_factor
     end
   end
 
+  def tour
```

Comment:
I tried to put this action on `users_controller` but when i try to perform a `PUT` request this redirects the request to `/teams`. How can i fix this and move this action to `users_controller`?